### DISCUSSION TITLE
Move to poetry only for development?

### DISCUSSION THREAD

**Discussion by thoroc** [rdeepak2002/reddit-place-script-2022#335] [Ideas]
Poetry support `pyproject.toml` with a version number that currently sits at 1.0.0. We could have a github action to trigger a `poetry version patch` on PR merge.

**Comment by ixi2101** [comment]
Yes big in favour of this. If we can add a log message that includes the version number it would make resolving issues so much easier.

**Comment by ixi2101** [comment]
Tagging #337 as we should move to a more poetry friendly layout if we do this
